### PR TITLE
Create boagen.md

### PR DESCRIPTION
**Pull request by deavmi**

### CODE REVIEW

**Comment by biilmann** [comment]
Need an actual body text with some information about boagen before we can merge this in.

Ideally something about what makes it special, why or when you would use boagen instead of any of the other static site generators listed...

**Comment by deavmi** [comment]
@biilmann Okay. Will update soon. :smile:

**Comment by deavmi** [comment]
Just added the **Why to use BoaGen over other static generators?**

**Comment by deavmi** [comment]
@biilmann Done. :smile:

File: source/projects/boagen.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+homepage: https://deavmi.github.io/BoaGen
+language: Bash
+license: GNU GPL v3.0+
+templates: None
+description: A Basic HTML Page Generator written in Bash.
+---
+
+##Why is BoaGen better than the others?
+
+It's so simple to use and also the best part ever is that BoaGen is a simple Bash script as compared to otehr static page/site generators which require the installation of Ruby, Pyhton and Ruby Gems. Think about it, what system doesn't have Bash?
```
Comment (Walkman100) [inline source/projects/boagen.md]: `otehr`

File: source/projects/boagen.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+##When to use Boagen?
+
+You should use BoaGen if you are the type of person taht needs a no frills basic HTML page to be generated and do not want to install anything else on your machine.
```
Comment (Walkman100) [inline source/projects/boagen.md]: `taht`

**Comment by deavmi** [comment]
**Credits go to:** Okay. Thank to @Walkman100 for pointing that out.

**Comment by deavmi** [comment]
Thanks ;D . Will it be on the site soon?



-----------------------
Sent from Deavmi
Keep on hacking...

Contact Deavmi via:

Pjuu: https://pjuu.com/deavmi
GNU Social: deavmi@quitter.se
Twitter: https://twitter.com/deavmi
Main mailbox: deavmi@openmailbox.org

**Comment by deavmi** [comment]
Oh I see it's on the site. Thanks. :)

**Pull request merged**